File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/WebPlayerConfig.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class WebPlayerConfig{

	@SerializedName("webPlayerActionsPorting")
	private WebPlayerActionsPorting webPlayerActionsPorting;

	public void setWebPlayerActionsPorting(WebPlayerActionsPorting webPlayerActionsPorting){
		this.webPlayerActionsPorting = webPlayerActionsPorting;
	}

	public WebPlayerActionsPorting getWebPlayerActionsPorting(){
		return webPlayerActionsPorting;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/WebPlayerConfig.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class WebPlayerConfig : Parcelable, Serializable {
    @SerializedName("webPlayerActionsPorting")
    var webPlayerActionsPorting: WebPlayerActionsPorting? = null
    override fun toString(): String {
        return "WebPlayerConfig{" +
                "webPlayerActionsPorting = '" + webPlayerActionsPorting + '\'' +
                "}"
    }
}